Project.toml

```diff
@@ -64,6 +66,8 @@ SparseArrays = "1.6"
 StaticArrays = "=1.9.3"
 Statistics = "1.6"
 StructTypes = "=1.10.0"
+Symbolics = "=5.30.3"
+SymbolicUtils = "2.0.2"
```

Comment:
You might need an `=` here too

Reply:
Alright. I will do it like` SymbolicUtils = "=2.0.2"`